views/default/object/plugin/full.php

```diff
@@ -111,7 +111,7 @@
 	$options['class'] = 'elgg-button elgg-button-cancel';
 	$options['text'] = elgg_echo('admin:plugins:deactivate');
 	if (!$can_activate) {
```

Comment:
a) active first with all dependencies met... remove a plugin
b) manual deactivation
c) some other reason... not very hard to come by though